<?xml version="1.0" encoding="utf-8"?>
<TestRun id="fad310a5-aa16-4532-b883-e3b6829c92a8" name="anna@cloudtest-win10 2023-01-22 23:59:15" runUser="cloudtest-win10\anna" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2023-01-22T23:59:15.0987318+00:00" queuing="2023-01-22T23:59:15.0987324+00:00" start="2023-01-22T23:59:08.2766792+00:00" finish="2023-01-22T23:59:15.1345216+00:00" />
  <TestSettings name="default" id="cc86c7b7-9196-466d-9d94-4aab174e4169">
    <Deployment runDeploymentRoot="anna_cloudtest-win10_2023-01-22_23_59_15" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="026b94dc-2e9c-4b9e-b594-87f5fc974603" testId="bbf7a954-76f9-1351-d799-dd3678204055" testName="Tester.AzureUtils.AzureTableDataManagerTests.AzureTableDataManager_UpsertTableEntryAsync" computerName="cloudtest-win10" duration="00:00:01.9550656" startTime="2023-01-22T23:59:14.7853769+00:00" endTime="2023-01-22T23:59:14.7854813+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="026b94dc-2e9c-4b9e-b594-87f5fc974603" />
  </Results>
  <TestDefinitions>
    <UnitTest name="Tester.AzureUtils.AzureTableDataManagerTests.AzureTableDataManager_UpsertTableEntryAsync" storage="c:\users\anna\orleans\test\extensions\testerazureutils\bin\debug\net7.0\tester.azureutils.dll" id="bbf7a954-76f9-1351-d799-dd3678204055">
      <Execution id="026b94dc-2e9c-4b9e-b594-87f5fc974603" />
      <TestMethod codeBase="C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\bin\Debug\net7.0\Tester.AzureUtils.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="Tester.AzureUtils.AzureTableDataManagerTests" name="AzureTableDataManager_UpsertTableEntryAsync" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="bbf7a954-76f9-1351-d799-dd3678204055" executionId="026b94dc-2e9c-4b9e-b594-87f5fc974603" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Completed">
    <Counters total="1" executed="1" passed="1" failed="0" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.5+1caef2f33e (64-bit .NET 7.0.2)&#xD;
[xUnit.net 00:00:01.27]   Discovering: Tester.AzureUtils (method display = ClassAndMethod, method display options = None)&#xD;
[xUnit.net 00:00:01.49]   Discovered:  Tester.AzureUtils (found 246 test cases)&#xD;
[xUnit.net 00:00:01.50]   Starting:    Tester.AzureUtils (parallel test collections = off, max threads = 4)&#xD;
[xUnit.net 00:00:03.56]   Finished:    Tester.AzureUtils&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="cloudtest-win10" outcome="Warning" timestamp="2023-01-22T23:59:14.9473741+00:00">
        <Text>Data collector 'Blame' message: All tests finished running, Sequence file will not be generated.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>